src/components/data-arcgis-layer-manager/data-arcgis-layer-manager-component.jsx

```diff
@@ -0,0 +1,30 @@
+import { setLayerOrder, setOpacity } from 'utils/arcgis-layer-manager-utils';
+
+const ArcgisLayerManager = ({ map, activeLayers }) => {
+  // Map prop is inherited from Webscene component
+  // reference: https://github.com/Esri/react-arcgis#advanced-usage
+  
+  const { layers } = map;
+  setLayerOrder(activeLayers, map);
+
+  layers.items.forEach(mapLayer => {
+    const setActive = activeLayers && activeLayers.some(activeLayer => activeLayer.title === mapLayer.title);
+    mapLayer.layers && mapLayer.layers.items && mapLayer.layers.items.forEach(layer => {
+      const setActive = activeLayers && activeLayers.some(activeLayer => activeLayer.title === layer.title);
+      setOpacity(layer, activeLayers);
+      layer.visible = !!setActive;
+      mapLayer.visible = !!setActive;
+    })
+    setOpacity(mapLayer, activeLayers);
+
+    // Set group layers as visible to allow sublayers to be visible
+    if (mapLayer.type === "group") {
```

Comment:
We also need this statement on the `featuredGlobe` (if we do not have it we are not able to see the `LabelsLayer` as it is a group layer).
It also makes our `initial-state` `activeLayers` to work properly (that happens when we set `mapLayer.visible = !!setActive;`) otherwise the layers will get their initial state from the Arcgis Online `scene` it self (all layers are visible by default there).

Reply:
Done :ok_hand: